packages/browser/src/Widget/Server/Workspace/SparqlWorkspaceQueryWindow.js

```diff
@@ -0,0 +1,304 @@
+import { SparqlEndpointResponseProvider } from '../SPARQL/Service/SparqlEndpointResponseProvider';
+import { SparqlQueryWindow } from '../SPARQL/View/SparqlQueryWindow';
+import * as itowns from 'itowns';
+import { Temporal3DTilesLayerWrapper } from '../../Temporal/Temporal';
+import { getUriLocalname } from '../../../URLUtil';
+
+/**
+ * The SPARQL query window class which provides the user interface for querying
+ * a SPARQL endpoint and displaying the endpoint response.
+ */
+export class SparqlWorkspaceQueryWindow extends SparqlQueryWindow {
+  /**
+   * Creates a SPARQL query window.
+   *
+   * @param {SparqlEndpointResponseProvider} sparqlProvider The SPARQL Endpoint Response Provider.
+   * @param {itowns.PlanarView} itownsView itowns view
+   * @param {object} configSparqlWidget The sparqlModule view configuration.
+   * @param {object} configSparqlWidget.queries Query configurations
+   * @param {object} configSparqlWidget.queries.title The query title
+   * @param {object} configSparqlWidget.queries.filepath The path to the file which contains the query text
+   * @param {object} configSparqlWidget.queries.formats Configuration for which visualizations are allowed
+   *                                                    with this query. Should be an object of key, value
+   *                                                    pairs. The keys of these pairs should correspond
+   *                                                    with the cases in the updateDataView() function.
+   */
+  constructor(sparqlProvider, itownsView, configSparqlWidget) {
+    super(sparqlProvider, itownsView, configSparqlWidget);
+
+    /** @type {Map<string,Temporal3DTilesLayerWrapper>} */
+    this.temporalWrappers = new Map();
```

Comment:
this attribute is not used inside the class it should be removed (in the example this is the user which is using it)